### ISSUE TITLE
Logits for entire output

### ISSUE THREAD

**Issue by Sanchit-404** [facebookresearch/llama#823]
Is there anyway i can get logits for all the generated tokens?

**Comment by QuantumBoIt** [comment]
I also have the same problem. How did you solve it?
